rest/route/subscription_route_test.go

```diff
@@ -0,0 +1,83 @@
+package route
```

Comment:
The existing naming convention says this file should be named "subscription_test.go"